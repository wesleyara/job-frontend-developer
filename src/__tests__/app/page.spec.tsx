import { render } from "@testing-library/react";
import Page from "~/app/page";
import { beforeEach, describe, expect, it } from "vitest";

describe("Page unit tests", async () => {
  let sut: JSX.Element;

  beforeEach(async () => {
    sut = await Page();
  });

  it("should mounted the page", () => {
    expect(sut).toBeDefined();
  });

  it("should render correctly", () => {
    const { container } = render(sut);

    expect(container).toMatchSnapshot();
  });
});
