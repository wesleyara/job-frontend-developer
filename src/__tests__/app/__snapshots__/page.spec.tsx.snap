// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Page unit tests > should render correctly 1`] = `
<div>
  <main
    class="h-[100vh] flex flex-col items-center justify-center bg-[#ebebeb]"
  />
</div>
`;
